tests/spec/core/webidl-spec.js

```diff
@@ -69,26 +69,104 @@ describe("Core - WebIDL", () => {
     });
   });
 
-  describe("records", () => {
-    it("handles record types", async () => {
-      const body = `
-        <section id="records">
-          <h2>Testing records</h2>
-          <pre class="idl">
-            dictionary Foo {
-              record&lt;DOMString, USVString> pass;
```

Comment:
Now I see no test for records 👀

It would be easier to review if the constructor part and the cleanup part is splitted.

Reply:
Yeah, I probably should have kept records - I thought those had been removed from IDL for some reason (wrong assumption). I'll add it back. 

As for the rest, I'm trying to get rid of WebIDL.html. It's full of errors and because of it's size (and because we never really look at it), it's not feasible to maintain.